### ISSUE TITLE
Request to Add PAT Token to Backend Environment

### ISSUE THREAD

**Issue by sahsisunny** [Real-Dev-Squad/website-backend#1459]
**Description:**
We need to add a Personal Access Token (PAT) to the Backend Environment to enable secure access to GitHub resources. This token will be used for various operations, such as fetching code, interacting with repositories, and automating tasks related to our GitHub repositories.

**Details:**
`GitHub Personal Access Token: GITHUB_PERSONAL_ACCESS_TOKEN`

**Route which needs PAT**
- /users?query=filterBy:UNMERGED_PRS+days:<Days> (to get list of inactive users)

**Steps to create GitHub PAT**

1. **Login to GitHub:**
   Ensure you are logged in to your GitHub account. You need to have the necessary permissions for the repositories you want to access.

2. **Access Your Account Settings:**
   Click on your profile picture in the upper-right corner of the GitHub website, then click on "Settings."

3. **Navigate to Developer Settings:**
   In the left sidebar, under "Settings," click on "Developer settings."

4. **Personal Access Tokens:**
   In the "Developer settings" menu, click on "Personal access tokens."

5. **Generate a New Token:**
   Click on the "Generate token" button.

6. **Token Settings:**
   Fill out the following information for your token:

   - **Token name:** Provide a descriptive name for your token, so you can easily identify its purpose.
   - **Expiration:** You can set an expiration date for added security or leave it empty for a token that doesn't expire.
   - **Select Scopes:** To create a read-only token, scroll down and under "repo," select at least the "public_repo" scope. If you need access to private repositories as well, select the "repo" scope.

7. **Generate Token:**
   Once you've configured the token settings, scroll down and click on the "Generate token" button.

8. **View and Copy Token:**
   After generating the token, GitHub will display it only once. Make sure to copy the token and save it in a secure location, such as a password manager. You won't be able to see this token again.

9. **Use the Token:**
   You can now use this token in your applications, scripts, or tools to access GitHub repositories with the permissions you specified during token creation. Remember to treat this token with care and never share it publicly or commit it to a repository.

10. **Optional: Review and Revoke Tokens:**
    To review or revoke your tokens, you can go back to the "Personal access tokens" section in your GitHub account settings. If you need to revoke a token for security reasons or if it's no longer needed, click on the token and select "Revoke" at the bottom of the token's detail page.

That's it! You now have a Personal Access Token that provides read-only access to your GitHub repositories. Remember to use it responsibly and keep it secure.

**Comment by ankushdharkar** [comment]
✅ Added 
![image](https://github.com/Real-Dev-Squad/website-backend/assets/1935403/b61d237a-4a40-4055-966f-de4fcd286a4f)
